__docs__/wonder-blocks-modal/one-pane-dialog.stories.tsx

```diff
@@ -282,7 +322,7 @@ export const WithAboveAndBelow: StoryComponentType = () => {
                 <OnePaneDialog
                     title="Single-line title"
                     content={
-                        <View style={{gap: sizing.size_160}}>
+                        <View style={{gap: sizing.size_160}} tabIndex={0}>
```

Comment:
This addresses the scrollable container accessibility issue in Chromatic. Odd that it just started failing -- maybe because it just now had changes, it prompted a new test run?